### PR TITLE
feat: improve MultisigRune.ratify

### PR DESCRIPTION
**Pull request by kratico**
- add `nonExecuting` to `MultisigRune.ratify`
- remove `MultisigRune.approve`

### CODE REVIEW

File: examples/multisig/basic.eg.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ $.assert($.array($.sizedUint8Array(32)), approvals)
 
 // Approve the proposal as Carol (final approval).
 await multisig
```
Comment (harrysolovay) [inline examples/multisig/basic.eg.ts:83]: Should we get rid of this intermediate non-executing ratify? Would simplify the example.

**Review by harrysolovay — COMMENTED** [review]
Shall we add some tsdoc comments to the members of `VirtualMultisig`?

**EDIT:**

Meant `MultisigRune`

**Comment by kratico** [comment]
> Shall we add some tsdoc comments to the members of `VirtualMultisig`?

I need to get familiar with [Proxies](https://wiki.polkadot.network/docs/learn-proxies)

Would it be ok to document `VirtualMultisig` in a separate PR?

**Comment by harrysolovay** [comment]
@kratico apologies, I meant `MultisigRune`

**Review by harrysolovay — CHANGES_REQUESTED** [review]
The params of `VirtualMultisigRune`'s `ratify` method should also change (ie. the addition of the `nonExecuting` param).

**Review by harrysolovay — APPROVED** [review]

**Pull request merged**